packages/near-sdk-js/package.json

```diff
@@ -46,6 +46,8 @@
     "commander": "^9.4.1",
     "eslint": "^8.20.0",
     "json-schema": "0.4.0",
+    "lodash": "^4.17.21",
+    "lodash-es": "^4.17.21",
```

Comment:
Can we depend only on lodash-es?

Reply:
yep, `lodash` is unsued, I will removed it

Reply:
removed